### PR TITLE
Remove Useless Renders

### PR DESCRIPTION
**Pull request by sgershuny**
• Halves number of coin renders on Main Page
• Third of coin renders on Swap page

### CODE REVIEW

File: src/components/coins/Coin/Coin.js [hunk 1]
```diff
@@ -0,0 +1,287 @@
+import React, { useEffect, useState } from "react";
```
Comment (lawlawlaw) [inline src/components/coins/Coin/Coin.js:1]: Is this a new file or copied from somewhere?

**Review by lawlawlaw — CHANGES_REQUESTED** [review]
You have added an empty file - src/components/coins/Coin/Coin.css.

File: src/components/coins/Coin/Coin.js (hunk 1 above)
Comment (sgershuny) [reply src/components/coins/Coin/Coin.js:1]: I have split the previous file coins.js into 2 files: CoinList which is the component that holds each of the Coin components.

This code here is the part of the previously named coins.js file that rendered each individual coin

**Review by lawlawlaw — APPROVED** [review]

**Review by lawlawlaw — APPROVED** [review]

**Review by lawlawlaw — CHANGES_REQUESTED** [review]
Tests are failing

**Pull request merged**